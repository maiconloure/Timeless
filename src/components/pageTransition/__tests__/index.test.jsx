import React from 'react';
import PageTransition from '../index';
import renderer from 'react-test-renderer';

describe('Snapshot', () => {
  it('renderer', () => {
    const tree = renderer.create(<PageTransition />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
