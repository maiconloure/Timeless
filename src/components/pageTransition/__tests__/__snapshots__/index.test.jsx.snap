// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot renderer 1`] = `
<div
  style={
    Object {
      "opacity": 0,
    }
  }
/>
`;
